_posts/2019-06-07-python-datascience-with-openfaas.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Serving Data Science Models with OpenFaaS
+# TODO
+description: "Build and deploy a name classification function with OpenFaaS [<--FUN SUGGESTIONS WELCOME]"
+date: 2019-06-01
+image: /images/multifile-python-function/pexels-book-stack-books-bookshop-264635.jpg
+categories:
+  - python
+  - coding
+  - examples
+  - datascience
+  - templates
+author_staff_member: lucas
+dark_background: true
+---
+
+Last month OpenFaaS was at [Kubecon Barcelona][kubecon-homepage], a frequent topic was deploying data science models to Kubernetes and, of course, can OpenFaaS help deploy models? In this post we will introduce a new function template aimed at Python data scientists and walk through a concrete example of deploying a [PyTorch][pytorch-homepage] model.
```

Comment:
> Last month OpenFaaS was at

`Last month I gave my first [KubeCon talk] on how to get PRs merged quickly in Open Source.`

Reply:
But the article isn't about that talk, it seems unrelated